data-import/src/main/resources/folijet/data-import/global/common-data-import.feature

```diff
@@ -41,20 +40,37 @@ Feature: Do Data Import Using Parameters
     * def createDate = response.fileDefinitions[0].createDate
     * def uploadedDate = createDate
 
-    ## Upload marc-file
-    Given path 'data-import/uploadDefinitions', uploadDefinitionId, 'files', fileId
-    And headers headersUserOctetStream
-    * print 'FilePath ---', filePathFromSourceRoot
+    Given path 'data-import/uploadUrl'
```

Comment:
is split files functionality enabled on the karate env by default?

Reply:
It will be going forward

Reply:
will data import succeed in this case since the common-data-import.feature has been changed to load file using s3 but for file processing regular fileProcessor will be used? Also considering that many of the tests were refactored to re-use this feature file.

Reply:
yes, but since the functionality is not enabled the regular fileProcessor will be used that looks for file locally. Or did I miss something?

Reply:
so it is enabled at the deployment/cluster level for the environment/cluster where karate tests regular autorun is performed, right?

Reply:
Correct

Reply:
thanks